### ISSUE TITLE
docker image doesn't recognize local-ip.co Let's Encrypt certs

### ISSUE THREAD

**Issue by mrjones-plip** [medic/cht-couch2pg#113]
When testing with this image, I saw I was getting an error when trying to contact my CouchDB instance on my localhost which was using a Let's Encrypt cert from https://local-ip.co :

```
curl https://192-168-68-26.my.local-ip.co/
curl: (60) SSL certificate problem: certificate has expired
```

Thinking what could cause this when the cert was most definitely not expired, I thought maybe the image was out of date. Then I did an `apt update && apt dist-upgrade` in the container, the error went away. I noted that `openssl` along with a lot of `python*` and `libpython*` packages were updated.

We should update the image on docker hub to account for this.

**Comment by mrjones-plip** [comment]
To followups:
* this issue could possibly causing complications in #112 
* I noticed the [docker hub image](https://hub.docker.com/r/medicmobile/cht-couch2pg)  is parented to the wrong GH repo. It uses [nomulex/cht-couch2pg](https://www.github.com/nomulex/cht-couch2pg) when it should use this one.

**Comment by mrjones-plip** [comment]
Fix will go live in #119
